Makefile

```diff
@@ -258,18 +260,23 @@ local-buildx: ## Build all container images necessary to run the whole operator
 	buildah manifest rm $(MARVELL_VSP_IMAGE)-manifest || true
 	buildah manifest create $(MARVELL_VSP_IMAGE)-manifest
 	buildah build --authfile /root/config.json --manifest $(MARVELL_VSP_IMAGE)-manifest --platform linux/amd64,linux/arm64 -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.mrvlVSP.rhel -t $(MARVELL_VSP_IMAGE)
+	buildah manifest rm $(INTEL_VSP_IMAGE)-manifest || true
+	buildah manifest create $(INTEL_VSP_IMAGE)-manifest
+	buildah build --authfile /root/config.json --manifest $(INTEL_VSP_IMAGE)-manifest --platform linux/amd64,linux/arm64 -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.IntelVSP.rhel -t $(INTEL_VSP_IMAGE)
```

Comment:
We need to set additional variables here as @sudhar-krishnakumar mentioned to ensure the fxp_linuxnetworking.pkg is installed.

i.e. 
`P4_NAME=fxp-net_linux-networking P4_DIR=fxp-net_linux-networking`
https://github.com/bn222/cluster-deployment-automation/blob/main/dpuVendor.py#L71